extensions/pipe-commands/assets/commands/text/accessibility/read-the-content.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# @raycast.title Read the content
+# @raycast.packageName Text Actions
+# @raycast.mode silent
+# @raycast.argument1 {"type": "text", "percentEncoded": false}
```

Comment:
Should`percentEncoded` be added here, or a copy error?

Reply:
@pernielsentikaer I think this param is not needed, just copy from other command to make sure it has the right param

Reply:
Awesome, could you remove it then and update the [changelog](https://developers.raycast.com/basics/prepare-an-extension-for-store#version-history) 😄